### PR TITLE
feat: [CDS-73382]:helm fetch metadata option

### PR DESCRIPTION
**Pull request by litvinovstas**
### Summary
Added option for helm chart, helm fetch metadata option

#### Screenshots
Screenshot 2023-07-03 at 14 36 49
Screenshot 2023-07-03 at 14 37 06

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by litvinovstas** [comment]
retrigger jest

File: src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmAdvancedStepSection.tsx [hunk 1]
```diff
@@ -144,6 +147,38 @@ function HelmAdvancedStepSection({
...
       </Layout.Horizontal>
+      <Layout.Horizontal
+        width={'90%'}
+        flex={{ justifyContent: 'flex-start', alignItems: 'center' }}
+        margin={{ top: 'medium' }}
+      >
+        {CDS_HELM_FETCH_CHART_METADATA_NG && deploymentType === ServiceDeploymentType.NativeHelm ? (
+          <>
+            <FormMultiTypeCheckboxField
+              key={isSkipVersioningDisabled.toString()}
```
Comment (chetanvekariya-harness) [inline src/modules/70-pipeline/components/ManifestSelection/ManifestWizardSteps/HelmAdvancedStepSection.tsx:158]: Why is key needed here?

**Review by chetanvekariya-harness — APPROVED** [review]

**Comment by litvinovstas** [comment]
retrigger cypress-rest

**Comment by litvinovstas** [comment]
retrigger cypress-cd

**Comment by litvinovstas** [comment]
retrigger cypress-pipeline

**Review by chetanvekariya-harness — APPROVED** [review]

**Comment by litvinovstas** [comment]
retrigger cypress-cd

**Review by sonaligoyal-harness — APPROVED** [review]

**Pull request merged**